modules/virt-about-node-labeling-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-about-node-labeling-obsolete-cpu-models_{context}"]
+= About node labeling for obsolete CPU models
+
+The {VirtProductName} Operator uses a predefined list of obsolete CPU models to ensure that a node supports only valid CPU models for scheduled VMs. By default, the following CPU models are eliminated from the list of labels generated for the node:
```

Comment:
I recommend putting a line break between the two sentences. Otherwise, it is easy for the reader to miss the second sentence.